WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -99,6 +99,12 @@ public void RecoverWallet(string walletName, string mnemonic, string password =
 		Global.WalletManager.AddWallet(keyManager);
 	}
 
+	[JsonRpcMethod("loadwallet", initializable: false)]
+	public void LoadWallet(string walletName)
```

Comment:
I don't think that's a good idea because loading a wallet can take a lot of time and rpc methods should return immediately. `Send` is a rare case because it performs an operation that can take a very few seconds but that could still be okay, imo.

Reply:
I agree with you. In many cases loading will take more time that the HttpTimeout of the client